apps/src/netsim/levels.js

```diff
@@ -153,6 +170,8 @@ levels.custom = {
   broadcastMode: false,
 
   // Packet header specification
+  addressFormat: '4',
+  packetCountBitWidth: 4,
```

Comment:
Going with sane defaults here for backwards-compatibility and because you shouldn't need to set either of these to zero/empty string - instead, you just omit the header type from the packet header spec, below.